### DISCUSSION TITLE
Opening links in a new window

### DISCUSSION THREAD

**Discussion by mmpadilla15** [thegetty/quire#452] [Get Help] [answered]
Hi all! On my museum's website, we open internal links in the same window and external links in a new window. Is there a way to do that in Quire? I've tried inserting the HTML code with link attribute target="_blank" but it doesn't seem to work.

**Comment by Erin-Cecele** [comment]
Hi @mmpadilla15. I saw you commented on #193. Just wanted to double-check and see if you tried that solution.

**Reply by mmpadilla15** [reply]
I did and I wasn't able to get it to work. But I just went through the steps again and saw that I saved the render-link file in wrong folder. I've moved it and it is working now! Yay!

But for internal links, do I need to change the markup like so:

`[Link](/essay)`

Or is there something else I need to do?

**Comment by Erin-Cecele** [comment, marked as answer]
Hi @mmpadilla15, glad to hear that you were able to get the external links working. As for internal links, if you are trying to link to a file called `essay.md` then the correct Markdown would be `[Link text](/essay/)`. Let me know if you are able to get it working correctly on your end.
